mipengine/node/node.py

```diff
@@ -9,6 +9,7 @@
 rabbitmq_socket_addr = node_config.rabbitmq.ip + ":" + str(node_config.rabbitmq.port)
 vhost = node_config.rabbitmq.vhost
 
+print("Creating the celery app...")
```

Comment:
logger?

Reply:
This code runs before the celery process is started and before the loggers etc are configured. I didn't want to initialize the loggers at this point, in fear that it could impact more things in the future (subsequent imports).

The first logs of the celery process don't have the formatted logger for the same reason, it's not configured yet, so I thought it made sense to keep it like that.

What do you think?

Reply:
well, it seems to me that the starting of a component is something you would normally want to log. 
"I didn't want to initialize the loggers at this point, in fear that it could impact more things in the future (subsequent imports)." It is not clear from that what's the danger? The logger should be very cheap and and easy to get anywhere in the app.

Reply:
Point taken. Made the appropriate changes.